pkg/codegen/internal/test/testdata/output-funcs/nodejs/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@pulumi/mypkg",
+    "version": "${VERSION}",
```

Comment:
@iwahbe this is here --^ ugly but hasn't caused compilation problems for tsc yet.

Reply:
It has no problems for `tsc`, but it will cause `npm install` to do nothing.